news/2019-05-06-project-loved-week-of-may-5th.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: post
+title: "Project Loved: Week of May 5th"
+date: 2019-05-06 16:00:00 +0000
+---
+
+We're a little late, but fear not, [Project Loved](https://osu.ppy.sh/community/forums/120) is still going strong! Check out the new picks that our captains have chosen, and vote for whether or not you'd like to see these maps enter the Loved category!
+
+![](/wiki/shared/news/banners/project-loved.jpg)
+
+Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! Every two weeks, we pick out a few beatmaps from each game mode that will get their shot at making it into the [Loved category](https://osu.ppy.sh/beatmapsets?s=8). From the maps we selected, you can now vote whether or not you want them to be Loved! Click on each map's header image in this post to be taken to its voting thread.
+
+Starting this week, we'll list on this news post whether a beatmapset will have any of its difficulties excluded from Loved, since it's caused some confusion in the past. The captains usually only exclude difficulties if they have major problems that prevent it from being played properly, but it's also common that multi-mode mapsets will have some game modes removed since the set isn't being nominated for those modes explicitly. If any difficulties of a map aren't being included in the Loved nomination, a note will be written below the map's image here (see Ryuu to Shoujo to Decoherence for an example).
```

Comment:
```suggestion
Starting this week, we'll list on this news post whether a beatmapset will have any of its difficulties excluded from Loved, since it's caused some confusion in the past. The captains usually exclude difficulties only if they have major problems that prevent them from being played properly, but it's also common that multi-mode mapsets will have some game modes removed, since the set isn't nominated for those modes explicitly. If any difficulties of a map aren't being included in the Loved nomination, a note will be written below the map's image here (see Ryuu to Shoujo to Decoherence for an example).
```